1.0/docs/devguide/templates.md

```diff
@@ -150,6 +148,9 @@ function defined on the host element.
 By default, the `filter` and `sort` functions only run when the array itself
 is mutated (for example, by adding or removing items).
 
+If you change the `filter` or `sort` function and want to re-process the data,
```

Comment:
These lines are incorrect. If you _change_ the `filter` or `sort` function, they'll be re-run automatically. You can use `render` to re-run the `filter` or `sort` function when some unrelated piece of data changes (for example, if your element has a `sortOrder` property that changes how the `sort` function works, you can call `render` when `sortOrder` changes.